docs/website/docs/plus/features/transformations/dbt-transformations.md

```diff
@@ -0,0 +1,225 @@
+---
+title: dlt+ dbt generator
+description: Generate dbt models automatically
+---
+
+The **dlt dbt generate** creates scaffolding for dbt projects using data ingested by dlt. It analyzes the pipeline schema and automatically generates staging and fact dbt models. By integrating with dlt-configured destinations, it automates code creation and supports incremental loading, ensuring that only new records are processed in both ingestion and transformation layers.
```

Comment:
I think we could point out here that this is the standalone version of the dbt generator.